src/test/resources/regressions/features/multisets/multiset-range-fail3.gobra

```diff
@@ -5,5 +5,5 @@ package pkg
 
 func foo() {
   //:: ExpectedOutput(assert_error:assertion_error)
-  assert 42 in mset[1..10]
+  assert 42 in mset[1 .. 10]
```

Comment:
I've considered this quite early, and I've revisited the problem and was finally able to fix the lexer so that it avoids this problem. In case it turns that this has introduced some hidden errors again, we can consider the other options again. Otherwise I would also implement a tailored error message for this.